<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>ProcessDetailDialog</class>
<widget class="QDialog" name="ProcessDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblPId">
        <property name="text">
         <string>PID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblUser">
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblState">
        <property name="text">
         <string>Estado:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblThreads">
        <property name="text">
         <string>Threads:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblParent">
        <property name="text">
         <string>PPID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCpuTime">
        <property name="text">
         <string>Tempo de CPU:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMemVirtual">
        <property name="text">
         <string>Memoria Virtual:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMemRam">
        <property name="text">
         <string>Memoria RAM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPriority">
        <property name="text">
         <string>Prioridade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFiles">
        <property name="text">
         <string>Arquivos abertos:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Fechar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
